tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/UpdateClaimedEpicsHandlerTests.cs

```diff
@@ -32,15 +35,32 @@ public void Setup()
         [Test]
         public async Task ShouldUpdateEpicsAsync()
         {
+            const int expectedEpicsCount = 1;
+
             var claimedEpics = new HashSet<IClaimedEpic>
             {
                 Mock.Of<IClaimedEpic>(e => e.EpicId == "Epic1" && e.StatusName == "Passed")
             };
 
+            _context.MockEpicRepository
+                .Setup(e => e.GetMatchingEpicIdsAsync(It.IsAny<IEnumerable<string>>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(expectedEpicsCount);
+
+            _context.MockSolutionEpicStatusRepository
+                .Setup(e => e.GetMatchingEpicStatusAsync(It.IsAny<IEnumerable<string>>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(expectedEpicsCount);
+
             var validationResult = await UpdateClaimedEpicsAsync(ValidSolutionId, claimedEpics).ConfigureAwait(false);
             validationResult.IsValid.Should().BeTrue();
+
             _context.MockSolutionRepository.Verify(s => s.CheckExists(ValidSolutionId, It.IsAny<CancellationToken>()),
                 Times.Once);
+
+            _context.MockEpicRepository.Verify(
+                e => e.GetMatchingEpicIdsAsync(It.IsAny<IEnumerable<string>>(), It.IsAny<CancellationToken>()), Times.Once);
+            _context.MockSolutionEpicStatusRepository.Verify(
+                e => e.GetMatchingEpicStatusAsync(It.IsAny<IEnumerable<string>>(), It.IsAny<CancellationToken>()), Times.Once);
```

Comment:
Here we need to verify the method is called with the expected list of strings. If you need a hand with this give me a shout.